packs/equipment/tacticians-helm.json

```diff
@@ -43,16 +43,26 @@
                 "selector": "warfare-lore",
                 "type": "item",
                 "value": 1
+            },
+            {
+                "key": "Note",
+                "outcome": ["success", "criticalSuccess"],
+                "predicate": [
+                    "encounter",
+                    {
+                        "not": "self:participant:own-turn"
+                    }
+                ],
+                "selector": "melee-strike-attack-roll",
+                "text": "On a successful @UUID[Compendium.pf2e.actionspf2e.Item.Reactive Strike], add one charge: @UUID[Compendium.pf2e.equipment-effects.Item.Effect: Tactician's Helm Charges]{Effect: Tactician's Helm Charges}",
+                "title": "{item|name}"
             }
         ],
         "size": "med",
         "stackGroup": null,
         "traits": {
             "rarity": "common",
-            "value": [
-                "invested",
-                "magical"
-            ]
+            "value": ["invested", "magical"]
```

Comment:
Was this extracted from Foundry or edited as a JSON?

Reply:
Hmm, I had extracted it from Foundry but I must have applied my IDE's formatting settings afterwards by mistake. I'll fix the style.